suite/suite.go

```diff
@@ -139,16 +140,24 @@ func Run(t *testing.T, suite TestingSuite) {
 	methodFinder := reflect.TypeOf(suite)
 	suiteName := methodFinder.Elem().Name()
 
-	for i := 0; i < methodFinder.NumMethod(); i++ {
-		method := methodFinder.Method(i)
-
-		ok, err := methodFilter(method.Name)
+	var matchMethodRE *regexp.Regexp
+	if *matchMethod != "" {
+		var err error
+		matchMethodRE, err = regexp.Compile(*matchMethod)
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "testify: invalid regexp for -m: %s\n", err)
 			os.Exit(1)
 		}
+	}
+
+	for i := 0; i < methodFinder.NumMethod(); i++ {
+		method := methodFinder.Method(i)
 
-		if !ok {
+		if !strings.HasPrefix(method.Name, "Test") {
```

Comment:
What about Benchmark Example and Fuzz?

Reply:
My point was about the ability to filter tests on `ExampleFoo`, `FuzzFoo` and `BenchmarkFoo`, where `Foo` is an identifier.

I'm sorry if it leads you to benchmark obvious

So here my remark was more about having something like this:

```go
		switch {
			case strings.HasPrefix(method.Name, "Test"),
				strings.HasPrefix(method.Name, "Example"),
				strings.HasPrefix(method.Name, "Fuzz"),
				strings.HasPrefix(method.Name, "Benchmark"):
			default:
				continue
		}
```

(naive implementation)